locale/fi/LC_MESSAGES/django.po

```diff
@@ -2279,10 +2279,9 @@ msgid "Requires"
 msgstr "Vaaditaan"
 
 #: exercise/templates/exercise/_user_results.html:55
-#, python-format
```

Comment:
Did you generate this file with `python manage.py makemessages`? I am a little surprised that the `python-format` flag has been removed since the string contains `%(deadline)s`. Just checking.